docs/reference/target-declaration.mdx

```diff
@@ -688,14 +684,15 @@ This parameter takes a non-negative integer and specifies the number of worker t
 
 <ShowIfs>
 <ShowIf ts >
-In the TypeScript target, the generated JavaScript program understands the following command-line arguments, each of which has a short form (one character) and a long form:
+In the TypeScript target, the generated JavaScript program understands the following command-line arguments, most of which have a short form (one character) and a long form:
 
-- `-f, --fast [true | false]`: Specifies whether to wait for physical time to match logical time. The default is `false`. If this is `true`, then the program will execute as fast as possible, letting logical time advance faster than physical time.
-- `-o, --timeout '<duration> <units>'`: Stop execution when logical time has advanced by the specified _duration_. The units can be any of nsec, usec, msec, sec, minute, hour, day, week, or the plurals of those. For the duration and units of a timeout argument to be parsed correctly as a single value, these should be specified in quotes with no leading or trailing space (e.g. '5 sec').
```

Comment:
The example `'5 sec'` seems helpful because otherwise the format would be unclear (for example, should I write something like `5sec` or `5.0 sec`?). Maybe we should keep that

Reply:
I think it may have been removed because this restriction no longer applies.

Reply:
Just realized this is with respect to the TypeScript runtime, the CLI args handling of which has not changed. So we probably should keep that line. Thanks for the catch.